src/components/PasswordSetterBox.js

```diff
@@ -52,7 +52,7 @@ class PasswordSetterBox extends Nimiq.Observable {
     }
 
     /**
-     * @param {?HTMLFormElement} $el
+     * @param {?HTMLFormElement | undefined} $el
```

Comment:
What's the difference between `| undefined` in the type and putting `$el` in square brackets?

Reply:
`[$el]` is an optional parameter which can be omitted. A parameter with `| undefined` can be set to `undefined` but not omitted. An optional parameter can not precede a non-optional parameter, which is why here it needs to be declared as `| undefined`.

In the old ts version, there was also another difference, which I now didn't observe anymore with the newer version: https://github.com/nimiq/keyguard/pull/461#discussion_r1045729111